core/network/src/protocol.rs

```diff
@@ -1226,7 +1226,7 @@ impl<B: BlockT, S: NetworkSpecialization<B>, H: ExHashT> Protocol<B, S, H> {
 					error
 				);
 				self.peerset_handle.report_peer(who.clone(), RPC_FAILED_REPUTATION_CHANGE);
-				Default::default()
+				StorageProof::empty()
```

Comment:
Why not implement `Default` for `StorageProof`?

Reply:
I don't think the concept of a "default proof" makes any sense.